packages/active-campaign-client/src/cognito.ts

```diff
@@ -0,0 +1,7 @@
+import { CognitoIdentityServiceProvider } from 'aws-sdk';
```

Comment:
This import is not working it should be added to package json or commented/deleted
```suggestion
// import { CognitoIdentityServiceProvider } from 'aws-sdk';
```